nusight2/src/client/components/chart/line_chart/view.tsx

```diff
@@ -32,11 +32,11 @@ export class LineChart extends Component<
 
     return (
       <>
-        <div className="bg-[#ddd] py-2 px-3">
+        <div className="bg-auto-surface-2 shadow shadow-md py-2 px-3">
```

Comment:
Shadow doesn't work as well in dark mode. This goes with my suggestion for the surface colors in the tailwind config (as the auto border will now be suitable in dark mode as well).

```suggestion
        <div className="bg-auto-surface-2 border-y border-auto py-2 px-3">
```